### PR TITLE
InvestorsModal now sortable by AmountInvested or UserAddress

### PR DESCRIPTION
**Pull request by kyrers**
This solves issue #953

A couple of notes:

1. This is my first pull request in Aelin. I tried to match my coding style to yours while keeping things simple. Please correct me if I did anything wrong and let me know about improvements;
2. I took the liberty of also allowing users to sort by investor address. I did this because the header was already sortable (but had no functionality) and because it can be helpful to look up investors.

### CODE REVIEW

**Comment by saeta-eth** [comment]
hey @kyrers, this is great! Thanks for contributing.

I ran into a small bug when I tried it in production mode. The investment amounts have the wrong order and don't follow the ordering rules you set in the code. I just checked the subgraph results and they work correctly. Inside the useAelinInvestors function, there is another orderBy function that is causing this issue.

Do you want to fix it?

File: src/components/pools/common/InvestorsModal.tsx [hunk 1]
```diff
@@ -145,22 +145,35 @@ const InvestorsTable = genericSuspense(
...
+      orderDirection: orderDirection,
       where: {
         poolAddress: pool.address,
       },
     })
 
-    const handleSort = () => {
-      if (orderDirection === OrderDirection.Desc) return setOrderDirection(OrderDirection.Asc)
-      setOrderDirection(OrderDirection.Desc)
+    const handleSort = (orderBy: Investor_OrderBy | undefined) => {
```
Comment (0xDmitry) [inline src/components/pools/common/InvestorsModal.tsx:160]: Maybe `newSortBy` instead of `orderBy` will be more clear?

**Review by 0xDmitry — APPROVED** [review]
Looks good! Only one minor suggestion

File: src/components/pools/common/InvestorsModal.tsx (hunk 1 above)
Comment (kyrers) [reply src/components/pools/common/InvestorsModal.tsx]: Makes sense. It's done here d57094d6663c1b4e5c15e1b1eff76f33284012d1

**Pull request merged**